### ISSUE TITLE
FPGA Board

### ISSUE THREAD

**Issue by jshield** [eigenco/frankenpi#1]
Hello, I saw you mentioned using a cheap Altera Cyclone IV Board, 

does it have a make/model, where did you source it from?

Cheers,

**Comment by eigenco** [comment]
It was/is one of the brandless what used to be cheap Chinese Cyclone IV boards from ebay. Unfortunately getting FPGAs is next to impossible at the moment due to chip shortage. Someone might occasionally still sell them, but the prices are usually excessive. I paid for my board something like 10 eur before covid, but now I've seen sellers asking over 200 eur for similar very simple boards. When chips become available again, I might design my own PCB for whatever is properly available and makes sense then, perhaps even a complete self contained FPGA board so a separate module would not be needed, but for the moment it seems one has to wait.

**Comment by AK6DN** [comment]
Are you running the Cyclone IV FPGA device out of spec? The Cyclone IV device has an absolute maximum limit of 4.2V on it's I/Os.
They are 3.3V I/O level compatible at most, they are NOT 5.0V I/O compatible.
The PC bus uses 5.0V I/O levels.
Normally one would use 5V/3.3V I/O translators on those signals (ie, like a TXB0104 device).

**Comment by eigenco** [comment]
Yes, the input voltage is out of spec. Never the less, it doesn't seem to be problem so far. I've run the board for 100s of hours without an issue so I will probably only make changes if I design a new board for broad audience, but so far when FPGAs are not available I didn't bother with it as software development didn't suffer.
